app/experimenter/nimbus-ui/src/components/AppLayoutWithSidebar/index.tsx

```diff
@@ -10,17 +10,55 @@ import Col from "react-bootstrap/Col";
 import Nav from "react-bootstrap/Nav";
 import { BASE_PATH } from "../../lib/constants";
 import classNames from "classnames";
+import { ReactComponent as ChevronLeft } from "./chevron-left.svg";
+import { ReactComponent as Cog } from "./cog.svg";
+import { ReactComponent as Layers } from "./layers.svg";
+import { ReactComponent as ChartArrow } from "./chart-arrow.svg";
+import { ReactComponent as Person } from "./person.svg";
+import { ReactComponent as Clipboard } from "./clipboard.svg";
+import { ReactComponent as NotAllowed } from "./not-allowed.svg";
+import { ReactComponent as AlertCircle } from "./alert-circle.svg";
 
 type AppLayoutWithSidebarProps = {
   testid?: string;
   children: React.ReactNode;
+  // TODO: Will be updated as part of EXP-464 and EXP-466
+  review?: {
+    invalidPages: string[];
+    ready: boolean;
+  };
 } & RouteComponentProps;
 
+const editPages = [
```

Comment:
Nice, I was still hesitant on needing name _and_ slug here at first but we need both in several places here so seems fine to me.